notifications/slack.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	defaultReleaseTemplate = `Release {{with .Cause}}{{if .User}}({{.User}}){{end}}{{end}} {{trim (print .Spec.ImageSpec) "<>"}} to {{with .Spec.ServiceSpecs}}{{range $index, $spec := .}}{{if not (eq $index 0)}}, {{if last $index $.Spec.ServiceSpecs}}and {{end}}{{end}}{{trim (print .) "<>"}}{{end}}{{end}}. {{with .Error}}{{.}}. failed{{else}}done{{end}}`
+	defaultReleaseTemplate = `Release {{with .Cause}}({{if .User}}{{.User}}{{else}}unknown{{end}}: {{if .Message}}{{.Message}}{{else}}<no message>{{end}}){{end}} {{trim (print .Spec.ImageSpec) "<>"}} to {{with .Spec.ServiceSpecs}}{{range $index, $spec := .}}{{if not (eq $index 0)}}, {{if last $index $.Spec.ServiceSpecs}}and {{end}}{{end}}{{trim (print .) "<>"}}{{end}}{{end}}. {{with .Error}}{{.}}. failed{{else}}done{{end}}`
```

Comment:
I'd say in the no-message case, just have `(paul)`, can drop the `<no message>` bit.